Source/utils/paths.cpp

```diff
@@ -100,7 +100,7 @@ const std::string &ConfigPath()
 const std::string &AssetsPath()
 {
 	if (!assetsPath)
-		assetsPath.emplace(AppPath() + "assets/");
+		assetsPath.emplace(BasePath() + "assets/");

```

Comment:
The intent here is to load it from the build folder. Changing this to use BasePath will cause the game to fail to launch if a recent version of the devilutionx.mpq ins't in the search paths.

Reply:
Actually `BasePath` was the same as `AppPath` unless set via CLI flags. Removed `AppPath` in https://github.com/diasurgical/devilutionX/pull/3770

Reply:
Yes, that is the point. It should be the path of the exe even when the CLI flag is given.

Reply:
But why? We do not know that `assets/` is next to the exe
`--data-dir` is mostly used by distros that place the data files in the custom likes

Reply:
> if a recent version of the devilutionx.mpq ins't in the search paths.

devilutionx.mpq is not looked up via this. It is only used for `assets/`

Reply:
We do, it's put there by the build process. `assets/` is only used when running the application from an IDE and not having smpq installed. It is not intended for use by the end user.

Reply:
I think you are confusing direct file access (which uses just a relative path) and `assets/` which each other. `assets/` is purely for developers. I guess this also explains why you where confused regarding the path where files should be placed for the iOS container.